lib/fog/compute/google/models/images.rb

```diff
@@ -68,6 +68,25 @@ def get(identity)
           new(data)
         end
 
+        def get_from_family(family)
+          # Search own project before global projects
+          all_projects = [service.project] + global_projects
+
+          data = nil
+          all_projects.each do |project|
+            begin
+              data = service.get_image_from_family(family, project).body
+              data[:project] = project
+            rescue Fog::Errors::NotFound
+              next
+            else
+              break
+            end
+          end
+          return nil if data.nil?
+          new(data)
+        end
```

Comment:
@plribeiro3000 You're talking about searching through Project collection via `get_project`? If so - I dunno how to make it work :/ In this It searches global image projects, in which you don't have access to the actual resource, observe:
```
[3] pry(main)> gce.get_project("dev-utility-xxxxxxxx")
=> #<Excon::Response:0x007f89f5fea390
 @body=
  {"kind"=>"compute#project",
   "id"=>"106685033xxxxxxxxxx",
   "creationTimestamp"=>"2015-04-08T01:59:07.888-07:00",
   "name"=>"dev-utility-xxxxxxxx",
[4] pry(main)> gce.get_project("centos-cloud")
Fog::Errors::Error: Required 'compute.projects.get' permission for 'projects/centos-cloud'
from /Users/temikus/Code/fog-devel/fog-google/lib/fog/google/shared.rb:183:in `build_excon_response'
```